leverage/modules/auth.py

```
@@ -117,11 +118,20 @@ def refresh_layer_credentials(cli):
+                roleName=sso_role,
+                accountId=account_id,
+                accessToken=cli.get_sso_access_token(),
+            )["roleCredentials"]
+        except ClientError as error:
+            if error.response["Error"]["Code"] == "ForbiddenException":
+                raise ExitError(
+                    40,
+                    f"User does not have permission to assume role [bold]{sso_role}[/bold]"
+                    " in this account.\nPlease check with your administrator or try"
+                    " running [bold]leverage aws configure sso[/bold].",
+                )
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)